### PR TITLE
Add docs redirects

### PR DESCRIPTION
**Pull request by anthonydeguzman**
**Proposed changes**:
Added redirects based off SEO recommendations

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/main/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: docs/netlify.toml [hunk 1]
```diff
@@ -202,3 +202,103 @@ to = "/docs/rasa/deploy/deploy-rasa/"
 [[redirects]]
 from = "/docs/rasa/action-server/deploy-action-server"
 to = "/docs/rasa/deploy/deploy-action-server/"
+
+[[redirects]]
+from = "/docs/rasa-x/changelog/rasa-x-changelog/"
+to = "/docs/rasa-enterprise/changelog/rasa-enterprise-changelog/"
```
Comment (m-vdb) [inline docs/netlify.toml:208]: not sure if this rule will work, it conflicts with [this one on rasa-website](https://github.com/RasaHQ/rasa-website/blob/111584e42f7a6ac8bd68dccc8a432c274c41c718/netlify.toml#L402-L404)

File: docs/netlify.toml [hunk 2]
```diff
@@ -202,3 +202,103 @@ to = "/docs/rasa/deploy/deploy-rasa/"
...
+
+[[redirects]]
+from = "/docs/rasa-enterprise/changelog/rasa-x-changelog/"
+to = "/docs/rasa-enterprise/changelog/rasa-enterprise-changelog/"
```
Comment (m-vdb) [inline docs/netlify.toml:212]: can you move all the rasa-enterprise redirects in the rasa-x repo? [here](https://github.com/RasaHQ/rasa-x/blob/main/docs/netlify.toml)

File: docs/netlify.toml [hunk 3]
```diff
@@ -202,3 +202,103 @@ to = "/docs/rasa/deploy/deploy-rasa/"
...
+from = "/docs/rasa-enterprise/annotate-conversations/"
+to = "/docs/studio/user-guide/annotation/"
+
+[[redirects]]
+from = "/docs/rasa-enterprise/0.24.1/installation-and-setup/integrated-version-control/"
+to = "/docs/rasa-enterprise/installation-and-setup/post-deploy-steps/set-up-ivc/"
+
+[[redirects]]
+from = "/docs/getting-started/demobot/"
+to = "/docs/"
```
Comment (m-vdb) [inline docs/netlify.toml:276]: I think this rule needs to be added to [rasa-website](https://github.com/RasaHQ/rasa-website/blob/main/netlify.toml) for it to work

File: docs/netlify.toml [hunk 4]
```diff
@@ -202,3 +202,103 @@ to = "/docs/rasa/deploy/deploy-rasa/"
...
+[[redirects]]
+from = "/docs/rasa/nlu/language-support/"
+to = "/docs/rasa/language-support/"
+
+[[redirects]]
+from = "/docs/rasa/api/tracker/"
+to = "/docs/rasa/action-server/sdk-tracker/"
+
+[[redirects]]
+from = "/docs/get_started_step1/"
```
Comment (m-vdb) [inline docs/netlify.toml:303]: I think this rule needs to be added to [rasa-website](https://github.com/RasaHQ/rasa-website/blob/main/netlify.toml) for it to work

**Review by m-vdb — CHANGES_REQUESTED** [review]
the `main` branch on `RasaHQ/rasa` doesn't receive updates anymore; can you point to the `documentation` branch instead? Your changes will be automatically deployed once the `documentation` branch receives new commits.

**Comment by anthonydeguzman** [comment]
Tried changing the base branch looks like I messed up this pull request. I'll open a different one

**Pull request closed without merging**